test/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ install(
     PATTERN "*.pb"
     PATTERN "tests_*.txt"
 )
-
+return()
```

Comment:
Is this required?

Reply:
this condition is required for windows as gtests are not enabled yet,  modified it accordingly

Reply:
so can it be under WIN flag, will it have any impact on other OSes